### PR TITLE
[FAB-17190] Add unit test for statecouchdb buildCommittersForNs, buildCommitters

### PR DESCRIPTION
**Pull request by DereckLuo**
Signed-off-by: Chongxin Luo <Chongxin.Luo@ibm.com>

#### Type of change

- Improvement (improvement to code, performance, etc)
- Test update

#### Description

- Added unit test for statecouchdb buildCommittersForNs.
- Added version nil check for statecouchdb updates commit.
- Added unit test for statecouchdb buildCommitters.
#### Related issues

[FAB-17190](https://jira.hyperledger.org/browse/FAB-17190)
[FAB-17191](https://jira.hyperledger.org/browse/FAB-17191)

### CODE REVIEW

**Comment by DereckLuo** [comment]
Both tests are targeting the same commit flow, therefore put it together under the same PR for review.
Will squash it before merge.

File: core/ledger/kvledger/txmgmt/statedb/statecouchdb/commit_handling_test.go
Comment (manish-sethi) [inline core/ledger/kvledger/txmgmt/statedb/statecouchdb/commit_handling_test.go]: instead of hardwired "1000", you can use "env.config.MaxBatchUpdateSize"

File: core/ledger/kvledger/txmgmt/statedb/statecouchdb/commit_handling_test.go
Comment (manish-sethi) [inline core/ledger/kvledger/txmgmt/statedb/statecouchdb/commit_handling_test.go:115]: Add one more assertion - assert.Equal(t, "ns", committers[1].namespace)

File: core/ledger/kvledger/txmgmt/statedb/statecouchdb/commit_handling_test.go
Comment (manish-sethi) [inline core/ledger/kvledger/txmgmt/statedb/statecouchdb/commit_handling_test.go]: It will be good to add maxBatch+1 number of keys to one of the namespaces and eventually check for three committers.

**Comment by DereckLuo** [comment]
@manish-sethi Thanks for the reviews, changes has been applied.

**Review by manish-sethi — DISMISSED** [review]

**Review by manish-sethi — APPROVED** [review]

**Pull request merged**
